website/src/main/tut/docs/rules/MissingFinal.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+title: MissingFinal
+---
+
+# MissingFinal
+
+_Since 0.5.8_
+
+This rule checks for or adds final modifier in the corresponding places.
+It has two use cases:
+- Check that descendants of a sealed types are final or sealed. 
```

Comment:
> Reports an error on classes that descend a sealed type but are not final.

BTW. Is there a reason why we shouldn't automatically add `final` to these cases?

Reply:
Yeah, I also did it initially :) But there's some imbalance if we add `final` to classes, but not `sealed` to traits, although they're almost identical from this point of view.